scripts/start.sh

```
@@ -6,25 +6,45 @@ echo "🎵 Starting A Lo Cubano Boulder Fest Development Server..."
+        echo "❌ Failed to install Vercel CLI."
+        echo "Try running: npm install -g vercel"
+        exit 1
+    fi
+    echo "✅ Vercel CLI installed successfully!"
 fi
 
-# Activate virtual environment
-echo "🔧 Activating virtual environment..."
-source venv/bin/activate
+# Ensure we're in the project root
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)